MdePkg/Include/Pi/PiStatusCode.h

```diff
@@ -966,27 +966,29 @@ typedef struct {
 /// These are shared by all subclasses.
 ///
 ///@{
-#define EFI_SW_EC_NON_SPECIFIC             0x00000000
-#define EFI_SW_EC_LOAD_ERROR               0x00000001
-#define EFI_SW_EC_INVALID_PARAMETER        0x00000002
-#define EFI_SW_EC_UNSUPPORTED              0x00000003
-#define EFI_SW_EC_INVALID_BUFFER           0x00000004
-#define EFI_SW_EC_OUT_OF_RESOURCES         0x00000005
-#define EFI_SW_EC_ABORTED                  0x00000006
-#define EFI_SW_EC_ILLEGAL_SOFTWARE_STATE   0x00000007
-#define EFI_SW_EC_ILLEGAL_HARDWARE_STATE   0x00000008
-#define EFI_SW_EC_START_ERROR              0x00000009
-#define EFI_SW_EC_BAD_DATE_TIME            0x0000000A
-#define EFI_SW_EC_CFG_INVALID              0x0000000B
-#define EFI_SW_EC_CFG_CLR_REQUEST          0x0000000C
-#define EFI_SW_EC_CFG_DEFAULT              0x0000000D
-#define EFI_SW_EC_PWD_INVALID              0x0000000E
-#define EFI_SW_EC_PWD_CLR_REQUEST          0x0000000F
-#define EFI_SW_EC_PWD_CLEARED              0x00000010
-#define EFI_SW_EC_EVENT_LOG_FULL           0x00000011
-#define EFI_SW_EC_WRITE_PROTECTED          0x00000012
-#define EFI_SW_EC_FV_CORRUPTED             0x00000013
-#define EFI_SW_EC_INCONSISTENT_MEMORY_MAP  0x00000014
+#define EFI_SW_EC_NON_SPECIFIC                    0x00000000
+#define EFI_SW_EC_LOAD_ERROR                      0x00000001
+#define EFI_SW_EC_INVALID_PARAMETER               0x00000002
+#define EFI_SW_EC_UNSUPPORTED                     0x00000003
+#define EFI_SW_EC_INVALID_BUFFER                  0x00000004
+#define EFI_SW_EC_OUT_OF_RESOURCES                0x00000005
+#define EFI_SW_EC_ABORTED                         0x00000006
+#define EFI_SW_EC_ILLEGAL_SOFTWARE_STATE          0x00000007
+#define EFI_SW_EC_ILLEGAL_HARDWARE_STATE          0x00000008
+#define EFI_SW_EC_START_ERROR                     0x00000009
+#define EFI_SW_EC_BAD_DATE_TIME                   0x0000000A
+#define EFI_SW_EC_CFG_INVALID                     0x0000000B
+#define EFI_SW_EC_CFG_CLR_REQUEST                 0x0000000C
+#define EFI_SW_EC_CFG_DEFAULT                     0x0000000D
+#define EFI_SW_EC_PWD_INVALID                     0x0000000E
+#define EFI_SW_EC_PWD_CLR_REQUEST                 0x0000000F
+#define EFI_SW_EC_PWD_CLEARED                     0x00000010
+#define EFI_SW_EC_EVENT_LOG_FULL                  0x00000011
+#define EFI_SW_EC_WRITE_PROTECTED                 0x00000012
+#define EFI_SW_EC_FV_CORRUPTED                    0x00000013
+#define EFI_SW_EC_INCONSISTENT_MEMORY_MAP         0x00000014
+#define EFI_SW_EC_MEMORY_TYPE_INFORMATION_CHANGE  0x00000015 // MU_CHANGE

```

Comment:
I thought this is the same as `EFI_SW_EC_INCONSISTENT_MEMORY_MAP`? Do we need this 0x15 definition for anything?

Reply:
Looks like it's used here for Implement MemoryTypeInformationChangeLib: https://github.com/microsoft/mu_plus/blob/005cfdd5b9942e98ff6218a7b06360f6b7606396/MsCorePkg/Library/MemoryTypeInformationChangeLib/MemoryTypeInformationChangeLib.c#L38

From spec:
> 0x0015-0x00FF
> Reserved for future use by this specification for Host Software class error codes.

I can break it out into its own commit since it seems unrelated to the addition of
> EFI_SW_EC_RELEASE_ASSERT

Reply:
No where in edk2 or in MU repos is EFI_SW_EC_INCONSISTENT_MEMORY_MAP consumed, interestingly.

Reply:
Yeah, the MU_CHANGE definition `EFI_SW_EC_MEMORY_TYPE_INFORMATION_CHANGE` should be replaced with `EFI_SW_EC_INCONSISTENT_MEMORY_MAP`. This macro was renamed during the process of updating the PI spec...

Reply:
Talked offline, `EFI_SW_EC_MEMORY_TYPE_INFORMATION_CHANGE` is defunct and can be removed. Will need to remove reference from MU_PLUS